integration-tests/src/server.js

```diff
@@ -20,26 +21,44 @@ const app = express()
 const server = http.createServer(app)
 server.setTimeout(serverTimeout)
 
+const firestore = new Firestore()
+
 const testTimeout = async timeout => {
   await delay(timeout)
   throw new Error(`Test timeout after ${timeout}ms`)
 }
 
+const reportTest = (testGroup, startTime, host, testName, errorMessage) => {
+  const endTime = new Date()
+  const testReport = { testGroup, runtimeInMilliseconds: endTime - startTime, status: errorMessage ? 'error' : 'success', testName, errorMessage, host }
```

Comment:
Perhaps `errorMessage` would be better named `error`, since it contains `message` as well as other fields.